matter_server/common/helpers/util.py

```diff
@@ -87,15 +87,20 @@ def parse_utc_timestamp(datetimestr: str):
     return datetime.fromisoformat(datetimestr.replace("Z", "+00:00"))
 
 
-def parse_value(name: str, value: Any, value_type: Type | str, default: Any = MISSING):
+def parse_value(
+    name: str, value: Any, value_type: Union[Type, str], default: Any = MISSING
```

Comment:
this is inside an annotation so would have been fine.

Reply:
Yeah, unrelated. The issue was the parsing of the actual annotations in this function itself.